Formula/pythran.rb

```diff
@@ -0,0 +1,116 @@
+class Pythran < Formula
+  desc "Ahead of Time compiler for numeric kernels"
+  homepage "https://pythran.readthedocs.io/"
+  url "https://files.pythonhosted.org/packages/2e/e9/bc3c16d683dbba5d34666b9861d1c1fdb07da0539655a3cf23a62e4cafa1/pythran-0.9.11.tar.gz"
+  sha256 "a317f91e2aade9f6550dc3bf40b5caeb45b7e012daf27e2b3e4ad928edb01667"
+  license "BSD-3-Clause"
+
+  keg_only <<~EOS
+    this formula is mainly used internally by other formulae.
+    Users are advised to use `pip` to install pythran
+  EOS
+
+  depends_on "gcc" => :test
+  depends_on "numpy"
+  depends_on "openblas"
+  depends_on "python@3.9"
+  depends_on "six"
+
+  resource "beniget" do
+    url "https://files.pythonhosted.org/packages/50/b9/7d864a50d5d2821686d4374400de45c8e0678e553657343f5f0ea6aff98d/beniget-0.3.0.tar.gz"
+    sha256 "062c893be9cdf87c3144fb15041cce4d81c67107c1591952cd45fdce789a0ff1"
+  end
+
+  resource "decorator" do
+    url "https://files.pythonhosted.org/packages/da/93/84fa12f2dc341f8cf5f022ee09e109961055749df2d0c75c5f98746cfe6c/decorator-4.4.2.tar.gz"
+    sha256 "e3a62f0520172440ca0dcc823749319382e377f37f140a0b99ef45fecb84bfe7"
+  end
+
+  resource "gast" do
+    url "https://files.pythonhosted.org/packages/83/4a/07c7e59cef23fb147454663c3271c21da68ba2ab141427c20548ae5a8a4d/gast-0.4.0.tar.gz"
+    sha256 "40feb7b8b8434785585ab224d1568b857edb18297e5a3047f1ba012bc83b42c1"
+  end
+
+  resource "networkx" do
+    url "https://files.pythonhosted.org/packages/b0/21/adfbf6168631e28577e4af9eb9f26d75fe72b2bb1d33762a5f2c425e6c2a/networkx-2.5.1.tar.gz"
+    sha256 "109cd585cac41297f71103c3c42ac6ef7379f29788eb54cb751be5a663bb235a"
+  end
+
+  resource "ply" do
+    url "https://files.pythonhosted.org/packages/e5/69/882ee5c9d017149285cab114ebeab373308ef0f874fcdac9beb90e0ac4da/ply-3.11.tar.gz"
+    sha256 "00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3"
+  end
+
+  def install
+    openblas = Formula["openblas"].opt_prefix
+    pythranrc = prefix/"config.pythranrc"
+    pythranrc.write <<~EOS
+      [compiler]
+      blas=openblas
+      include_dirs=#{openblas}/include
+      library_dirs=#{openblas}/lib
+    EOS
+
+    xy = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
+
+    resources.each do |r|
+      r.stage do
+        system "python3", *Language::Python.setup_install_args(libexec)
+      end
+    end
+
+    system "python3", *Language::Python.setup_install_args(libexec)
```

Comment:
As note, this installs vendored `boost` and `xsimd`.

ArchLinux manually deletes the directories after install:
https://github.com/archlinux/svntogit-community/blob/packages/python-pythran/trunk/PKGBUILD#L25
```
  rm -r "$pkgdir"/usr/lib/python*/site-packages/pythran/{boost,xsimd} # Remove bundled boost and xsimd
```

There also appears to be a way to build without it, but requires more complicated steps. The 1st command is from https://github.com/serge-sans-paille/pythran/pull/733 but not sure about 2nd command:
```
python setup.py build_py --no-boost --no-xsimd
python setup.py install ... --skip-build
```